software/verification/README.md

```diff
@@ -0,0 +1,32 @@
+# Verifying installed control sequences using CDL
+
+A python based open-loop verification software to test whether the control sequences installed in HVAC controllers meet the required specification.
+The reference sequences to be tested must be present in the [Modelica Buildings Library](https://github.com/lbl-srg/modelica-buildings/) and
```

Comment:
I don't think the sequences must be in the Buildings library; this would be a huge requirement. Rather, they need to be present in the form of a Modelica model that conforms to the CDL standard (in order for the json file that has the input/output list to be generated)